guidelines/goals.md

```diff
@@ -0,0 +1,71 @@
+# Goals and Non-Goals
+
+This document defines the purpose of the Message Format Working Group (MFWG)
+and informs the decisions about the scope and the priorities of its efforts.
+
+## Goals
+
+The primary task of the MFWG is to develop an industry standard for the
+representation of localizable message strings. The design goals are listed
+below.
+
+ 1. Express complex grammatical features, such as plurals, genders, and
+    inflections, catalogued in the CLDR.
+
+ 2. Express other variance in translation, due to other linguistic and regional
+    features not listed in the CLDR, as well as due to the presentation media,
+    context, circumstance, and other factors.
+
+ 3. Represent structured data alongside translations, like markup, comments,
+    and metadata.
+
+ 4. Ensure interoperability with existing interchange formats, in particular
+    with XLIFF.
+
+ 5. Ensure that machine translation and translation memory solutions can be
+    used for authoring translations.
```

Comment:
If you meant "authoring messages" instead of "authoring translations", then this makes sense.  Otherwise, as it is written, it sounds like an l10n Translation Management System (TMS) concern that falls under item 5 of Non-goals ("build a l10n framework").

Reply:
I think we should be concerned about how the standard can be represented in TMS-es, and how well it integrates with them. For instance, I'd like us to consider how to structurally mark up placeholders in translation units so that machine translation can learn to omit them.

Reply:
+1 I think if we build a standard that cannot be used in the common tool, nobody will adopt it. Ideally, we should aim for out of the box support so that we don't have to wait many years for commercial solutions to get up to speed.

Reply:
As it's phrased right now, the goal no. 5 focuses on MT and TM. @nbouvrette's comment makes me think that we should make it broader, and mention the general integration with CAT/TMS toolchains. Something like:

> Ensure that the standard can be integrated into existing TMS and CAT toolchains.

Reply:
That suggested new phrasing sounds good, and I do agree with the reasoning for the new phrasing.